src/swf/SWFFile.ts

```diff
@@ -212,12 +210,18 @@ module Shumway.SWF {
         this._decompressor = Inflate.create(true);
         // Parts of the header are compressed. Get those out of the way before starting tag parsing.
         this._decompressor.onData = this.processFirstBatchOfDecompressedData.bind(this);
+        this._decompressor.onError = function (error) {
+          throw new Error(error);
```

Comment:
I'm pretty sure just throwing some error here isn't what we really want to do. No need to fix it now, but please add a TODO. We probably need to store the error on the FileLoader and call an error callback on the Loader, which can then do a `throwError` to content.